Update system-requirements.de.md

Ernestopheles opened the pull request:
Den Abschnitt Hosting-Konfiguration ergänzt (Kopie vom gleichnamigen Abschnitt unter Contao Installation mit der Kommandozeile).

Denn das muss ja immer gemacht werden, deshalb würde ich es hierher setzen. Dann kann man den Abschnitt unter Contao Installation mit der Kommandozeile wegnehmen.

Ernestopheles committed:
Update system-requirements.de.md

Den Abschnitt Hosting-Konfiguration ergänzt (Kopie vom gleichnamigen Abschnitt unter Contao Installation mit der Kommandozeile).

Denn das muss ja immer gemacht werden, deshalb würde ich es hierher setzen. Dann kann man den Abschnitt unter Contao Installation mit der Kommandozeile wegnehmen.

xchs commented on docs/manual/installation/system-requirements.de.md:
```diff
@@ -134,6 +134,17 @@ FastCGI-Setup zu konfigurieren. Durch die Verwendung von
 
+### Hosting-Konfiguration
+
+In Contao befinden sich alle öffentlich erreichbaren Dateien im Unterordner `/web` der Installation. Setze das 
```
@fritzmg Habt ihr eine Konvention festgelegt, wie Ordnernamen ausgezeichnet werden sollen? Im Manual findet man diese beiden Schreibweisen:
* `/web`
* `web`

Ich persönlich würde ja überall die unter Unix/Linux übliche Notation `web/` verwenden, aber das ist auch nur meine persönliche Präferenz.

fritzmg replied:
Sollten wir noch festlegen. In der dev doku ist es `web/` (evt. aber auch noch nicht durchgängig).

Ernestopheles replied:
okidoki

Ernestopheles committed:
Update system-requirements.de.md

web/ durch web ersetzt.

fritzmg commented on docs/manual/installation/system-requirements.de.md:
```diff
@@ -134,6 +134,17 @@ FastCGI-Setup zu konfigurieren. Durch die Verwendung von
 
+### Hosting-Konfiguration
+
+In Contao befinden sich alle öffentlich erreichbaren Dateien im Unterordner `web` der Installation. Setze das 
```
```suggestion
In Contao befinden sich alle öffentlich erreichbaren Dateien im Unterordner `web/` der Installation. Setze das 
```

fritzmg commented on docs/manual/installation/system-requirements.de.md:
```diff
@@ -134,6 +134,17 @@ FastCGI-Setup zu konfigurieren. Durch die Verwendung von
+Wurzelverzeichnis (Document Root) der Installation über das Admin-Panel des Hosting-Providers auf diesen 
+Unterordner und richte bei dieser Gelegenheit noch eine Datenbank ein.
+
+Beispiel: `example.com` zeigt auf das Verzeichnis `www/example/web`
```
```suggestion
Beispiel: `example.com` zeigt auf das Verzeichnis `/www/example/web`
```
Hier ist die absolute Pfadangabe schon richtig.

Ernestopheles committed:
Update system-requirements.de.md

Sorry, den "/" für root hatte ich versehentlich entfernt - ich glaube ich brauch 'ne Pause...

Ich persönlich finde web/ nicht so schön, schließlich verwenden wir bei Pfadangaben auch keinen abschließenden slash. Aber ist halt Geschmackssache.

fritzmg commented on docs/manual/installation/system-requirements.de.md:
```diff
@@ -134,6 +134,17 @@ FastCGI-Setup zu konfigurieren. Durch die Verwendung von
+Beispiel: `example.com` zeigt auf das Verzeichnis `/www/example/web`
+
+{{% notice note %}}
+Pro Contao-Installation wird deshalb eine eigene (Sub)Domain benötigt.
```
Man könnte hier noch erwähnen, dass es grundsätzlich empfehlenswert ist, unterschiedliche Web Applikationen auf Domainebene zu trennen.

xchs replied:
Meinst Du auf **Verzeichnisebene**?

fritzmg replied:
Nein, prinzipiell auf Domainebene.

xchs commented on docs/manual/installation/system-requirements.de.md:
```diff
@@ -134,6 +134,17 @@ FastCGI-Setup zu konfigurieren. Durch die Verwendung von
 [`fastcgi_finish_request()`](https://www.php.net/manual/en/function.fastcgi-finish-request.php) kann Contao 
 Hintergrundaufgaben (wie die Indizierung des Seiteninhalts) ausführen, ohne dass der Browser auf die Antwort wartet.
 
```
```suggestion


```

Ernestopheles commented:
Also sowas wie public_html/main/web für Contao und
public_html/myapplication/... für eine andere Applikation?

Am 08.11.19 um 18:39 schrieb Fritz Michael Gschwantner:
>
> *@fritzmg* commented on this pull request.
>
> ------------------------------------------------------------------------
>
> In docs/manual/installation/system-requirements.de.md
> <https://github.com/contao/docs/pull/160#discussion_r344288781>:
>
> > @@ -134,6 +134,17 @@ FastCGI-Setup zu konfigurieren. Durch die Verwendung von
>  [`fastcgi_finish_request()`](https://www.php.net/manual/en/function.fastcgi-finish-request.php) kann Contao 
>  Hintergrundaufgaben (wie die Indizierung des Seiteninhalts) ausführen, ohne dass der Browser auf die Antwort wartet.
>  
> +### Hosting-Konfiguration
> +
> +In Contao befinden sich alle öffentlich erreichbaren Dateien im Unterordner `web/` der Installation. Setze das 
> +Wurzelverzeichnis (Document Root) der Installation über das Admin-Panel des Hosting-Providers auf diesen 
> +Unterordner und richte bei dieser Gelegenheit noch eine Datenbank ein.
> +
> +Beispiel: `example.com` zeigt auf das Verzeichnis `/www/example/web`
> +
> +{{% notice note %}}
> +Pro Contao-Installation wird deshalb eine eigene (Sub)Domain benötigt.
>
> Man könnte hier noch erwähnen, dass es grundsätzlich empfehlenswert
> ist, unterschiedliche Web Applikationen auf Domainebene zu trennen.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/contao/docs/pull/160?email_source=notifications&email_token=ABZWBO2IKB232YR7MJ5W3RDQSWP4RA5CNFSM4JKZ5T6KYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCK6K2VI#pullrequestreview-314355029>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABZWBO6PS365ELRJODC3HGLQSWP4RANCNFSM4JKZ5T6A>.
>

fritzmg commented:
> Also sowas wie public_html/main/web für Contao und public_html/myapplication/... für eine andere Applikation?

* `www.example.org` => bspw. Contao
* `shop.example.org` => bspw. Magento
* `blog.example.org` => bspw. WordPress
* `api.example.org` => bspw. irgendeine andere Applikation
* etc.

Ernestopheles commented:
Also genau gesagt sieht es dann z.B. so aus:

domain/public_html/main/web für Contao

und

domain/public_html/myapplication/... für eine andere Applikation.

Das ist streng genommen eine Trennung auf Verzeichnisebene, jedoch
empfiehlt z.B. mein Hoster das Verzeichnis public_html zu verwenden,
u.a. weil Systemdateien des Hosters dort abgelegt werden.

Am 08.11.19 um 18:47 schrieb Fritz Michael Gschwantner:
>
> *@fritzmg* commented on this pull request.
>
> ------------------------------------------------------------------------
>
> In docs/manual/installation/system-requirements.de.md
> <https://github.com/contao/docs/pull/160#discussion_r344291812>:
>
> > @@ -134,6 +134,17 @@ FastCGI-Setup zu konfigurieren. Durch die Verwendung von
>  [`fastcgi_finish_request()`](https://www.php.net/manual/en/function.fastcgi-finish-request.php) kann Contao 
>  Hintergrundaufgaben (wie die Indizierung des Seiteninhalts) ausführen, ohne dass der Browser auf die Antwort wartet.
>  
> +### Hosting-Konfiguration
> +
> +In Contao befinden sich alle öffentlich erreichbaren Dateien im Unterordner `web/` der Installation. Setze das 
> +Wurzelverzeichnis (Document Root) der Installation über das Admin-Panel des Hosting-Providers auf diesen 
> +Unterordner und richte bei dieser Gelegenheit noch eine Datenbank ein.
> +
> +Beispiel: `example.com` zeigt auf das Verzeichnis `/www/example/web`
> +
> +{{% notice note %}}
> +Pro Contao-Installation wird deshalb eine eigene (Sub)Domain benötigt.
>
> Nein, prinzipiell auf Domainebene.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/contao/docs/pull/160?email_source=notifications&email_token=ABZWBOY6KXI5QNGIW6YYPVDQSWQ2ZA5CNFSM4JKZ5T6KYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCK6L3HI#discussion_r344291812>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABZWBO3KPB7NVIM73OKG7VTQSWQ2ZANCNFSM4JKZ5T6A>.
>

Ernestopheles commented:
Das wäre also eine Subdomain je weiterer Applikation und sieht dann im
Dateisystem z. B. wie folgt aus:

www.example.org => example.org/public_html/main/web für Contao

shop.example.org => example.org/public_html/shop/web für bspw. Magento

...

Am 08.11.19 um 18:50 schrieb Fritz Michael Gschwantner:
>
>     Also sowas wie public_html/main/web für Contao und
>     public_html/myapplication/... für eine andere Applikation?
>
>   * |www.example.org| => bspw. Contao
>   * |shop.example.org| => bspw. Magento
>   * |blog.example.org| => bspw. WordPress
>   * |api.example.org| => bspw. irgendeine andere Applikation
>   * etc.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/contao/docs/pull/160?email_source=notifications&email_token=ABZWBO6VTE5YC4JHC62UEZTQSWRFZA5CNFSM4JKZ5T6KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEDS3SVA#issuecomment-551926100>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABZWBO35BHVPZUD2OGTXO4LQSWRF3ANCNFSM4JKZ5T6A>.
>

fritzmg commented:
> Das ist streng genommen eine Trennung auf Verzeichnisebene

Indirekt ja, weil du ja nicht zwei verschiedene roots für die selbe Domain festlegen kannst.
